README.md

```diff
@@ -28,7 +28,7 @@ the [WASM wrapper](https://github.com/docknetwork/crypto-wasm).
       - [Terminology](#terminology)
       - [Examples](#examples)
         - [Selective disclosure](#selective-disclosure)
-        - [BBS signature over varying number of messages](#bbs-signature-over-varying-number-of-messages)
+        - [BBS signature over null-valued messages](#bbs-signatures-over-null-valued-messages)
```

Comment:
Using null-values is just one way to deal with it as the example shows other way as well. That section is trying to solve the problem of varying number of messages.